Assets/MRTK/Tests/PlayModeTests/GltfTests.cs

```diff
@@ -98,8 +100,32 @@ public IEnumerator TestGltfCustomAttributes()
             int temperature = gltfObject.accessors[temperatureIdx].count;
             Assert.AreEqual(100, temperature);
         }
+        
+        [UnityTest]
+        public IEnumerator TestGltfCustomAttributesData() {
+            // Load glTF
```

Comment:
Small nit to match brace style in the rest of the file:

```suggestion
        public IEnumerator TestGltfCustomAttributesData()
        {
            // Load glTF
```